pkg/shell/cockpit-docker.js

```diff
@@ -370,6 +370,24 @@ function render_container (client, $panel, filter_button, prefix, id, container)
     var added = false;
     if (!tr.length) {
         var img_waiting = $('<div class="waiting">');
+        var btn_delete = $('<button class="btn btn-danger pficon pficon-close btn-delete">').
+            on("click", function() {
+                var self = this;
+                $(this).hide().
+                    siblings("div.waiting").show();
+                client.rm(id, true).
+                    fail(function(ex) {
+                        shell.show_unexpected_error(ex);
+                        $(self).show().
+                            siblings("div.waiting").show();
+                    });
+                return false;
+            });
+        var wrapped = $('<div class="button-wrap-tooltip">').
```

Comment:
Could you add a comment about why this is necessary? About why the tooltip isn't added directly to the button?